boot/initramfs20dirs.go

```diff
@@ -71,7 +71,7 @@ func setInitramfsDirVars(rootdir string) {
 	InitramfsUbuntuBootDir = filepath.Join(InitramfsRunMntDir, "ubuntu-boot")
 	InitramfsUbuntuSeedDir = filepath.Join(InitramfsRunMntDir, "ubuntu-seed")
 	InstallHostWritableDir = filepath.Join(InitramfsRunMntDir, "ubuntu-data", "system-data")
-	InstallHostFDEDataDir = filepath.Join(InstallHostWritableDir, "var/lib/snapd/device/fde")
+	InstallHostFDEDataDir = dirs.SnapFDEDirUnder(InstallHostWritableDir)
```

Comment:
Maybe a :TODO:UC20: that we want some helper here that does not use InstallHostWritableDir outside of the install mode?

Reply:
I've added `SnapFDEDir` for this purpose., I think we can pick between the two in the relevant code as needed.